### DISCUSSION TITLE
Package pdfx Error: CreationDate is not properly supported

### DISCUSSION THREAD

**Discussion by sorashi** [tectonic-typesetting/tectonic#1058] [Q&A] [answered]
When using the pdfx package to create a PDF/A-2u output, with tectonic I get the following error:
```
Package pdfx Error: CreationDate is not properly supported
```

Minimal example:
```tex
\documentclass[12pt,a4paper]{report}
\usepackage{polyglossia}
\setmainlanguage{czech}
\setotherlanguages{english}
\usepackage[a-2u]{pdfx}

\begin{document}
    Hello
\end{document}
```

When I use latexmk, I get the same error. But with this latexmk configuration
```
$pdf_mode = 5;
$xelatex = "xelatex -interaction=nonstopmode -synctex=1 -shell-escape %O %S";
$xdvipdfmx = "xdvipdfmx -z 9 -V 7 -o %D %O %S";
$dvi_mode = 0;
$postscript_mode = 0;
```
the document is typeset correctly without errors.

What can I do to use tectonic with pdfx?

**Comment by vlasakm** [comment, marked as answer]
Does [this workaround](https://github.com/tectonic-typesetting/tectonic/issues/838#issuecomment-1279979430) from #838 work?

**Reply by sorashi** [reply]
Yes, it works. Sorry, I don't know how I missed that in my search.
